pkg/virtctl/imageupload/imageupload.go

```diff
@@ -549,9 +549,8 @@ func createPVCSpec(size, storageClass, accessMode string, blockVolume bool) (*v1
 		spec.AccessModes = []v1.PersistentVolumeAccessMode{v1.PersistentVolumeAccessMode(accessMode)}
 	}
 
-	if blockVolume {
-		volMode := v1.PersistentVolumeBlock
-		spec.VolumeMode = &volMode
+	if volumeMode != "" {
+		spec.VolumeMode = &v1.PersistentVolumeMode(volumeMode)
```

Comment:
I think we need a validation here.

Reply:
validate it in `parseArgs` for both accessMode and volumeMode, so it fails immediately if the value is not as expected.